src/frontend/static/icons/Hipster_CartIcon.svg

```diff
@@ -1 +1,69 @@
-<svg xmlns="http://www.w3.org/2000/svg" id="Layer_1" data-name="Layer 1" viewBox="0 0 20 20"><defs><style>.cls-1{fill:#b4b2bb}</style></defs><title>Hipster</title><g><path d="M7.76,14.3a1.1,1.1,0,1,0,1.1,1.1A1.1,1.1,0,0,0,7.76,14.3Zm5.5,0a1.1,1.1,0,1,0,1.1,1.1A1.1,1.1,0,0,0,13.26,14.3ZM7.85,12.51l0-.06.49-.9h4.1a1.1,1.1,0,0,0,1-.57l2.12-3.85-.95-.53h0L14,7.7H7.29L6.77,6.6,6.25,5.5H4.46V6.6h1.1l2,4.17-.75,1.35a1.19,1.19,0,0,0-.13.53,1.1,1.1,0,0,0,1.1,1.1h6.6v-1.1H8A.14.14,0,0,1,7.85,12.51Z" class="cls-1"/><path d="M10,2a8,8,0,1,1-8,8,8,8,0,0,1,8-8m0-2A10,10,0,1,0,20,10,10,10,0,0,0,10,0Z" class="cls-1"/></g></svg>
```

Comment:
The `Hipster_CartIcon.svg` file contains changes because the top-right cart icon has a new look.